README.md

```diff
@@ -1,24 +1,201 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <br/>
+  <h1>Motor's Reservation</h1>
 
-* Ruby version
+</div>
 
-* System dependencies
+## 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [Schema](#ER-diagram)
+  - [🛠 Built With ](#-built-with-)
+  - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+  - [Kanban Board](#kanban)
+  - [Video Presentation](#video-presentation)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Run Tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+- [🔭 Future Features:](#-future-features)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# Motor's Reservation <a name="about-project"></a>
 
-* How to run the test suite
+> The Backend for the Motor's Reservation app. Built using Ruby on Rails.
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- ER diagram"-->
 
-* Deployment instructions
+# Database Schema <a name="ER-diagram"></a>
```

Comment:
The title of the project is Motorhub. please change it. and Write a **meaningfull description of the project: Like this:** 
The Motorhub - Online Motor Reservation System is a web application designed to provide users with the ability to reserve, view, and manage motors such as cars and bikes reservation for purchases. The system aims to streamline the process of finding and booking motors from various locations, making it convenient for both users and motor providers.

Reply:
done